config/betterquesting/DefaultQuests/Quests/AndSoItBegins-AAAAAAAAAAAAAAAAAAAAAA==/CanYouHeartheCar-AAAAAAAAAAAAAAAAAAAB5A==.json

```diff
@@ -31,7 +31,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Tired of waiting since your carrots grow sooooo slow?\n\nWell I\u0027ve got an idea, bone meal will help you grow food in seconds. So, chop down some trees and I will trade you some bone meal."
+      "desc:8": "Tired of waiting since your crops grow sooooo slow?\n\nBone meal will help you to grow food in seconds. \n\nChop down some trees to get a bit of bone meal as a reward."
```

Comment:
carrots need to stay carrots. you missed the context.

Reply:
Thank you. Fixed.